### ISSUE TITLE
"Download failed" Error

### ISSUE THREAD

**Issue by tomere9** [argosopentech/argos-translate#380]
**tl;dr:**
1. Problem in downloading the language package programmatically by the library -> raises a runtime error: "Download failed for..."
2. Another possible problem: the language packages names that the library expects are different that the names of the downloaded packages.

**Elaborated:**
I tried copying the example code for Python, supplied in the main GitHub page:
```
import argostranslate.package
import argostranslate.translate

from_code = "en"
to_code = "es"

# Download and install Argos Translate package
argostranslate.package.update_package_index()
available_packages = argostranslate.package.get_available_packages()
package_to_install = next(
    filter(
        lambda x: x.from_code == from_code and x.to_code == to_code, available_packages
    )
)
argostranslate.package.install_from_path(package_to_install.download())

# Translate
translatedText = argostranslate.translate.translate("Hello World", from_code, to_code)
print(translatedText)
# '¡Hola Mundo!'

```
and now I get an error: "Download failed for English → Spanish."

I tried to debug it, and it seems that there is a strange behavior when attempting to install the language package:
When I looked at this line:
`argostranslate.package.install_from_path(package_to_install.download())
`
and then inside the '_download()_' method, I saw that the method tries to find a path '_filepath_' to the en->es package:
```
filename = argospm_package_name(self) + ".argosmodel"
...
filepath = settings.downloads_dir / filename

```
and it didn't find this '_filepath_', and so it calls the '_get_from_' method to download the data:
`data = networking.get_from(self.links)
`
yet it returned data = None, and that's what raised the error.

I then looked into the '_get_from_' method, where it seems to attempt downloading the missing en->es package from the links supplied in the arguments, and it somehow fails to do that by its own. However, when I open the "https://..." [link](https://pub-dbae765fb25a4114aac1c88b90e94178.r2.dev/v1/translate-en_es-1_0.argosmodel) in my browser , the browser succeeds to install the package (in my user downloads folder...). So it is not clear why the package couldn't be installed automatically at the first place.

That is my main problem, as it prevents even the sample script from working, but further investigation reveals another problem:

When I looked at the name of the downloaded package, it was: '_translate-en_es-1_0.argosmodel_', but looking back at the _filename_ that the '_download()_' method tried to find, it was directed to a package named '_translate-en_es.argosmodel_' - so even after downloading the package to the right folder, the '_filepath_' won't be found, as the downloaded package's name is different than the searched one. From what I understand this lead to installing the package every time the function is called, even if there is a relevant package already installed (but with an unexpected name to the script).
